doc/index.rst

```diff
@@ -12,12 +12,13 @@ Table of Contents
 
 .. toctree::
    :maxdepth: 2
+   :glob:
 
    Overview <overview>
    Installation and Setup <install>
-   Getting Started with Qiskit Terra <getting_started_with_qiskit_terra>
    The IBM Q Provider <the_ibmq_provider>
```

Comment:
This is in qiskit/basics so you can remove that as well.

Reply:
Oops, yeah I removed the file, just missed the index entry. Done